less/mixins.less

```diff
@@ -1 +1,13 @@
-// Mixins in here
\ No newline at end of file
+.flex {
+    display: flex;
+}
+
+.btn {
+    color: white;
+    background-color: @eastern-blue;
+    border-radius: 15%;
+    width: 100%;
+    height: 15%;     
+    padding-top: 5%;    
+    text-align: center;
+  }
```

Comment:
Please,  use a parametric mixin for the button.
Review TK https://learn.lambdaschool.com/fsw/module/rec1hru3bo6l0uxn2 on this topic. If you still have any questions, let me know